tests/e2e/statefulsets.go

```diff
@@ -312,23 +317,23 @@ var _ = ginkgo.Describe("statefulset", func() {
 			ginkgo.By("CNS_TEST: Running for vanilla k8s setup")
 			scParameters = nil
 			storageClassName = "nginx-sc-parallel"
+			scSpec := getVSphereStorageClassSpec(storageClassName, scParameters, nil, "", "", false)
+			sc, err := client.StorageV1().StorageClasses().Create(ctx, scSpec, metav1.CreateOptions{})
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			defer func() {
+				err := client.StorageV1().StorageClasses().Delete(ctx, sc.Name, *metav1.NewDeleteOptions(0))
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}()
+
 		} else {
-			storageClassName = defaultNginxStorageClassName
+			storageClassName = storagePolicyName
 			ginkgo.By("Running for WCP setup")
 			profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyName)
 			scParameters[scParamStoragePolicyID] = profileID
 			// create resource quota
-			createResourceQuota(client, namespace, rqLimit, storageClassName)
+			//createResourceQuota(client, namespace, rqLimit, storageClassName)
```

Comment:
If the createResourceQuota call is not needed - you can remove it

Reply:
done